hooks/rnc-format-validate.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+# assert the trang command exists
+if ! [ -x "$(command -v trang)" ]; then
+  echo 'trang command not found; please install'
+  exit 1
+fi
+
+# iterate through changed rnc files
+for rncfile in $(git diff --cached --name-only | grep -E '\.rnc$')
+do
+  rngfile="${rncfile//+(*\/|.*)}.rng"
+  trang "$rncfile" "$rngfile" && trang "$rngfile" "$rncfile"
+  rm -f "$rngfile"
+  git add "$rncfile"
```

Comment:
Here we add the commit so it doesn't fail, and so auto-fix the formatting of the file(s).